### ISSUE TITLE
Lähtötietokohteet

### ISSUE THREAD

**Issue by SannaJokela** [sykefi/kaavatietomalli#94]
Kaavakohteiden liittyminen lähtötietokohteeseen ei ole mallissa mietitty tunnuksen kirjaamista pitemmälle. Tämän voisi toteuttaa tietojärjestelmätasolla ja linkittää lähtötietotaulun mahdollisia geometrioita kaavakohteeseen.

**Comment by ilkkarinne** [comment]
Joo, tästä keskusteltiin AK-YK -projektissa, ja päädyttiin siihen, että Lähtötietokohde-luokkaa ei tuoda Kaavatietomalliin ainakaan tässä vaiheessa, vaan lähtötiedot linkitetään vain aineistotasolla. Malli ei ota kantaa miten rakenteisista ja viitattavista osista lähtötietoaineistot koostuvat. Kaavakohde-luokassa on tosiaan attribuutti liittyvanLahtotietokohteenTunnus[0..*]:URI, jolla kaavakohde voidaan linkittää mihin tahansa lähtötietoaineiston objektiin, jolla on URI-muotoinen tunnus.
